packages/botonic-plugin-google-translate/src/google-translate-api-service.ts

```diff
@@ -7,34 +7,76 @@ export interface LanguageDetection {
   confidence: number
 }
 
+export interface Translation {
```

Comment:
this should be added to [core](https://github.com/hubtype/botonic/blob/master/packages/botonic-core/src/types/index.d.ts#L71)
@marcrabat one option to avoid polluting core with the types of plugins is using [TS declaration merging](https://www.typescriptlang.org/docs/handbook/declaration-merging.html) so that Input.translations is added in the plugin, not in core. But then if we add a plugin for another translator engine we should move it again to core, so it's up to you

Reply:
yes, I think it would be better to import the intial declaration from core and extending it directly in the plugin, in favour of not pollution core with plugin definitions

Reply:
personally I prefer to add it to core because:
* it works as "advertising" for the botonic-core users that we have a gtranslate plugin
* we (or 3rd party plugins)can reuse it for future translation plugins
but ymmv :-)

Reply:
pretty convincing, go for it 😜